tests/unit/test_column_noise.py

```diff
@@ -454,7 +490,7 @@ def test_miswrite_ages_flips_negative_to_positive():
 
 
 @pytest.mark.slow
-def test_write_wrong_digits_robust(dummy_dataset):
+def test_write_wrong_digits_robust(dummy_dataset, fuzzy_checker: FuzzyChecker):
```

Comment:
I don't feel strongly, but I think we could add more assertions here. For example a column-level assertion for each column (instead of only asserting at the character-index-level) and an assertion across all columns.

In the old scheme, there was no advantage to adding these, but now the advantage would be sample size.

Reply:
There is a jira ticket for this: https://jira.ihme.washington.edu/browse/MIC-4794